server/router_test.go

```diff
@@ -1,27 +1,61 @@
 package server_test
 
 import (
-	"net/http"
-	"testing"
-
 	"github.com/gorilla/mux"
 	"github.com/runatlantis/atlantis/server"
 	. "github.com/runatlantis/atlantis/testing"
+	"net/http"
+	"testing"
 )
 
 func TestRouter_GenerateLockURL(t *testing.T) {
-	queryParam := "queryparam"
-	routeName := "routename"
-	atlantisURL := "https://example.com"
+	cases := []struct {
```

Comment:
I moved all your tests into a single case. No real reason other than it was more concise. The way you had it was fine too.